### PR TITLE
Setup ci

### PR DESCRIPTION
**Pull request by rahulporuri**
This PR adds the relevant files to setup CI on travis and appveyor.
The PR tests the package on 3 platforms - 2.7, 3.5 and 3.6.

Note that the files have been adapted from pyface. For example, the toolkits portion of the `etstool` module and CI config files have been removed as the scimath package doesn't use/need them.

### CODE REVIEW

**Comment by rahulporuri** [comment]
Sweet. It looks like Travis automatically found the config file and started builds. I'm not sure how to get appveyor to work.

File: etstool.py [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+from shutil import rmtree, copy as copyfile
+from tempfile import mkdtemp
+from contextlib import contextmanager
+
+import click
+
+supported_runtimes = ['2.7', '3.5', '3.6']
+
+dependencies = {
+    "numpy",
```
Comment (rkern) [inline etstool.py]: And `scipy`?

Comment (rahulporuri) [reply etstool.py]: Ahh. Thanks! Corrected. I just made that change in `setup.py`, how did I forget!

**Comment by rahulporuri** [comment]
merging for now. shall address appveyor later.

**Pull request merged**